CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.14.3 - 2nd May 2019 (released)
+
+### Added
+
+- Added validatePartnerName function
+
```

Comment:
I missed this before: Note that we haven't released these and shouldn't detail every build in here, just the final release that actually goes to prod, which will probably be 1.14.0. These builds would be 1.14.0.0, 1.14.0.1 etc. The 3rd part is for bug fix releases rather than builds.

Reply:
In order to test our functions, we need to publish this package, and in order to publish, we need to up-version.

Reply:
You're right, this is a limitation.